Assets/AnkrSDK/Runtime/MirageAPI/MirageID/Helpers/MirageEnvironment.cs

```diff
@@ -0,0 +1,68 @@
+using System.Collections.Generic;
+
+namespace AnkrSDK.MirageAPI.MirageID.Helpers
+{
+	public static class MirageEnvironment
+	{
+		private static readonly Dictionary<EnvironmentType, EnvSetting> Environments = CreateEnvironments();
+
+		private static EnvSetting _currentEnv;
+
+		private const string CreateUserEndpoint = "/Users";
+		private const string WalletEndpoint = "/Wallet";
+		private const string NftTransactionEndpoint = "/NftTransaction";
+		private const string NFTTransactionDataEndpoint = "/GetTransactionData";
+		private const string TransferNFTEndpoint = "/TransferNft";
+
+		public static string TokenEndpoint => _currentEnv.AuthUrl + "/realms/mirage-id/protocol/openid-connect/token";
+		public static string LogoutEndpoint => _currentEnv.AuthUrl + "/auth/realms/mirage-id/protocol/openid-connect/logout";
+		public static string CreateUserURL => _currentEnv.MirageIdUrl + CreateUserEndpoint;
+		public static string WalletInfoURL => _currentEnv.MirageIdUrl + WalletEndpoint;
+		public static string TransferNFTURL =>  _currentEnv.MirageIdUrl + NftTransactionEndpoint + TransferNFTEndpoint;
+		public static string NFTTransactionDataURL =>  _currentEnv.MirageIdUrl + NftTransactionEndpoint + NFTTransactionDataEndpoint;
+		public static string CreateNFT =>  _currentEnv.MirageIdUrl + "NftTransaction";
```

Comment:
Consider encapsulating different APIs under different entities, because the number of APIs will continue to grow.